core/conventional-commits/__tests__/conventional-commits.test.js

```diff
@@ -333,6 +333,44 @@ describe("conventional-commits", () => {
       expect(rootChangelogContent).toMatchSnapshot("root");
     });
 
+    it("should contain default header when custom header is not passed", async () => {
+      const cwd = await initFixture("fixed");
+      await gitTag(cwd, "dragons-are-awesome1.0.0");
+      const [pkg1] = await getPackages(cwd);
+
+      await pkg1.set("changed", 1).serialize();
+      await gitAdd(cwd, pkg1.manifestLocation);
+      await gitCommit(cwd, "fix: A second commit for our CHANGELOG");
+
+      // update version
+      await pkg1.set("version", "1.0.1").serialize();
+      const leafChangelog = await updateChangelog(pkg1, "fixed", {});
+      const changeLog = await getFileContent(leafChangelog);
+      expect(changeLog.indexOf("Change Log")).toBe(2);
+    });
+
+    it("should contain custom changelog header when passed", async () => {
+      const customHeader = "ChangeLog";
+      const cwd = await initFixture("fixed");
+      await gitTag(cwd, "dragons-are-awesome1.0.0");
+
+      const [pkg1] = await getPackages(cwd);
+
+      // make a change in package-1
+      await pkg1.set("changed", 1).serialize();
+      await gitAdd(cwd, pkg1.manifestLocation);
+      await gitCommit(cwd, "fix: A second commit for our CHANGELOG");
+
+      // update version
+      await pkg1.set("version", "1.0.1").serialize();
+      const leafChangelog = await updateChangelog(pkg1, "fixed", {
+        changelogHeader: customHeader,
```

Comment:
We also need a test from the version command that validates the `--changelog-header` option is being passed to the conventional commits utility, modeled after the [`--changelog-preset` option test](https://github.com/lerna/lerna/blob/91d9d3e597db6578b3be7f8ee88d1246ad0f80c5/commands/version/__tests__/version-conventional-commits.test.js#L250).